tests/unit/test_categories.py

```diff
@@ -72,10 +72,12 @@ def test_category_assignment(transaction: Transaction):
         ):
             # The sign of the category_assignment could be wrong for Transfer Transactions
             assert abs(transaction.amount) == pytest.approx(
-                abs(total_amount), abs=0.01
+                abs(total_amount), abs=TOLERANCE_AMOUNT
             ), (transaction, category_assignment)
         else:
-            assert transaction.amount == pytest.approx(total_amount, abs=0.01), (
-                transaction,
-                category_assignment,
-            )
+            pass
+            # TODO: it looks like category assignment amount contain weights, not actual amounts
```

Comment:
interesting. at least it's the actual amount on my side.

Reply:
I was surprised, too, but I don't have a better explanation for what I saw. 

**Transaction:**
```
WithdrawTransaction(id=22259,
                    reconciled=True,
                    amount=Decimal('273.1'),
                    description=***,
                    datetime=***,
                    notes='***',
                    account=5861,
                    payee=16880,
                    original_currency='UAH',
                    original_amount=Decimal('273.1'),
                    original_exchange_rate=Decimal('1.0'))
```
**Category Assignment:**
```
[
    (12290, Decimal('40.0')), 
    (21142, Decimal('113.1')), 
    (19942, Decimal('200.0'))
]
```
![2024-06-27_08-53](https://github.com/ileodo/moneywiz-api/assets/979567/2287679c-ea27-45db-93e2-e4cec4512341)